app/components/app_patient_page_component.html.erb

```diff
@@ -17,6 +17,31 @@
   <%= render AppPatientDetailsComponent.new(patient:, session:) %>
 <% end %>
 
+<% if gillick_assessment_applicable? %>
+  <%= render AppCardComponent.new(heading: "Gillick assessment") do %>
+    <% if gillick_assessment_recorded? %>
+      <%= govuk_summary_list(classes: "app-summary-list--no-bottom-border nhsuk-u-margin-bottom-0") do |summary_list| %>
+        <%= summary_list.with_row do |row| %>
+          <%= row.with_key { "Are they Gillick competent?" } %>
+          <%= row.with_value { patient_session.gillick_competent ? "Yes, they are Gillick competent" : "No" } %>
+        <% end %>
+        <%= summary_list.with_row do |row| %>
+          <%= row.with_key { "Details of your assessment" } %>
+          <%= row.with_value { patient_session.gillick_competence_notes } %>
+        <% end %>
+      <% end %>
+    <% else %>
+      <%= form_for :consent,
+        url: session_patient_manage_consents_path(session, patient_id: patient.id, section: @section, tab: @tab),
+        method: :post,
+        builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+        <%= f.govuk_submit "Assess Gillick competence", secondary: true,
+          name: "consent", value: "self_consent" %>
+      <% end %>
```

Comment:
⛳🏌️‍♂️ Untested:

```suggestion
    <%= govuk_button_link_to(
      "Assess Gillick competence",
      session_patient_manage_consents_path(session, patient_id: patient.id, section: @section, tab: @tab),
      secondary: true,
      name: "consent", value: "self_consent" # Does this work?...
    ) %>
```

I'm curious if that works, the `name/value` bits.

The only advantage to a `govuk_button_link_to` is just in code aesthetics as it more clearly communicates that "this is a self-contained form with no fields that just POSTs."

Reply:
Almost... I think it needs `govuk_button_to`, which is a wrapper around [`button_to`](https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to). I've updated here: [807f2b5](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/1344/commits/807f2b59d02813baf6c2411cf18732136a03fa43).

Reply:
Oh yeah, that’s what I mean to write!